src/Frontend/Modules/Slideshow/Layout/Templates/Detail.html.twig

```diff
@@ -25,11 +25,11 @@
           <div class="flexsliderwrap{{ gallery.id }}">
             <ul class="slides">
               {% for _itr_0 in slideshow %}
-              <li data-thumb="/src/Frontend/Files/_itr_0/thumbnails/{{ _itr_0.filename }}">
+              <li data-thumb="/src/Frontend/Files/slideshow/thumbnails/{{ _itr_0.filename }}">
```

Comment:
What's up with this var naming? `_itr_0.filename` 

Shouldn't it be something like `{{ slideshow.filename }}`